src/figuresClasses.ts

```diff
@@ -27,16 +33,16 @@ export class Triangle implements Figure {
   }
 
   getArea(): number {
-    const s: number = (this.a + this.b + this.c) / 2;
+    const square: number = (this.a + this.b + this.c) / 2;
```

Comment:
it is not a square - its a semiPerimeter

Reply:
sorry...)
done